test/spec/modules/nobidBidAdapter_spec.js

```diff
@@ -0,0 +1,167 @@
+import { expect } from 'chai';
```

Comment:
Your test coverage falls below the 80% test coverage minimum requirement.

![image](https://user-images.githubusercontent.com/13972794/64187686-e99cf400-ce25-11e9-8c00-5dc6135e1faf.png)

I think if you add a couple tests around `getUserSyncs` it should get above 80%.

If you have the time maybe try to cover some other lines/functions as well.

You can generate and view your test coverage by running the following:

```
gulp test-coverage
gulp view-coverage
```

We are cutting a new Prebid release tomorrow (Wednesday September 4th) around 1:00 PM PST.

If you get the test coverage up before then I will make sure to get it in the release of PBJS `2.31.0`!

Reply:
Done too.